ee/clickhouse/views/events.py

```diff
@@ -46,7 +46,8 @@ def _get_people(self, query_result: List[Dict], team: Team) -> Dict[str, Any]:
     def _query_events_list(
         self, filter: Filter, team: Team, request: Request, long_date_from: bool = False, limit: int = 100
     ) -> List:
-        limit_sql = f"LIMIT {limit + 1}"
+        limit += 1
+        limit_sql = f"LIMIT %(limit)"
```

Comment:
no need for f here! The parameter is specified by %()s so it should be `%(limit)s`